submission-folder/liyuan.ts

```diff
@@ -0,0 +1,114 @@
+/**
+ * HOMEWORK
+ * rename this file to *your name*.ts
+ * Translate this JS code into valid TypeScript that follows good practices,
+ * following instructions in the comments
+ */
+
+// replace all `var` declarations appropriatly
+
+const NINE_THOUSAND : number = 9000
```

Comment:
no need to explicitely assign type if it's implied by TypeScript already

Reply:
fixed in c1a1105cebd79ab481debf3da9c66a071f307d76